### ISSUE TITLE
+armor

### ISSUE THREAD

**Issue by Sritterbush** [Arx-Game/arxcode#246]
**Is your feature request related to a problem? Please describe.**
Currently it is difficult for a player to be sure which armor they have on, and which category of armor that it is in. While outright errors in descriptions are rare, I still recall a person wearing two sets of armor desced as pants that were assigned to different slots.

**Describe the solution you'd like**
A command that displays what armor items a person is wearing and in which slots.

**Describe alternatives you've considered**
Integrating this information into existing code, either through +combatstats or as part of the wear/worn system. Decided either of these would be very distracting and make the output too long.

**Additional context**
Not my first time talking about this idea, though I don't know what happened to the original mention from ages ago. I'm not asking others to do this, just vetting the idea before I get started. This is my first original bit of code I'm proposing adding to Arx, and will be attempting to write it myself.

**Comment by Zironic** [comment]
Just so you know what you're getting into. The game does not currently have an actual paperdoll system. Instead of the game keeping track of what you are wearing where, what is currently happening is that each time you equip an item, that item checks if there's too many other items of the same type already equiped.

https://github.com/Arx-Game/arxcode/blob/bfba68f617e308c4474ac7d426550505c5d69ce9/typeclasses/wearable/wearable.py#L92

**Comment by Sritterbush** [comment]
Yeah, and I know Tehom may rewrite that, so I might have to do this twice if I do it now. I accept the challenge, seems a function people would use that will teach me a good bit about the code.

**Comment by dreamsofgold** [comment]
You can still integrate it into the old code if you use the tags the objects have as slots in your paperdoll system.

**Comment by TehomCD** [comment]
Working on a paperdoll system is kind of what ground my crafting refactor to a halt: there was a lot of moving parts, particularly with making it work with the outfit and fashion systems. You don't really have to get into that so much: you could only check for items that have armor values and where they're worn
